client/packages/android/app/src/main/java/org/openmsupply/client/CertWebViewClient.java

```diff
@@ -200,7 +199,8 @@ public boolean shouldOverrideUrlLoading(WebView view, WebResourceRequest request
         // will not only reload, but will open the URL in a browser tab
         // for local URLs we don't want this to happen!
         Uri url = request.getUrl();
-        if(url.toString().startsWith(this.nativeApi.getServerUrl())) {
+        if(url.toString().startsWith("https://127.0.0.1:" + nativeApi.DEFAULT_PORT)
```

Comment:
What if the server is not local, if app is in client mode and connecting to another server ?

Reply:
Oh yes right, I guess this wouldn't work then. The case this change was covering, was that I had a debugUrl on my local machine and the `request.getUrl()` was `https://127.0.0.1`. Could ignore this case for now or add some debugUrl check...

Reply:
I reverted this change in [f3bce62](https://github.com/openmsupply/open-msupply/pull/1739/commits/f3bce6257e549d40c485a6ebf25b466ba031a654)